README.md

```diff
@@ -221,18 +222,18 @@ will be used:
 * For display, the `demo-card` component will be called
 * For edit, the `demo-card-editor` component will be called
 
-In display mode, the following attrs will be provided:
-
-* `data`, the data payload for this card
-* `editAction`, an action for toggling this card into edit mode
-
-In edit mode, the following attrs will be provided:
-
-* `data`, the data payload for this card
-* `saveAction`, an action accepting new data for the card payload, then saving
-  that data and toggling this card into display mode
-* `cancelAction`, an action toggling this card to display mode without saving
-  any changes.
+The component will be provided with the following `attrs`:
+
+  * `data`, the data payload for this card
+  * `editCard`, an action for toggling this card into edit mode (this action is a no-op if the card is already in edit mode)
+  * `removeCard`, an action for removing this card
+  * `saveCard`, an action accepting new data for the card payload, then saving
```

Comment:
Should these be split out as before, or at least be notated that `saveCard` for instance is not always present?